src/dmd/dsymbolsem.d

```diff
@@ -2365,7 +2365,7 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             if (auto te = ed.memtype.isTypeEnum())
             {
                 EnumDeclaration sym = cast(EnumDeclaration)te.toDsymbol(sc);
-                if (!sym.memtype || !sym.members || !sym.symtab || sym._scope)
+                if (!sym.memtype || (!sym.members && !sym.isSpecial()) || !sym.symtab || sym._scope)
```

Comment:
This line is hard to understand at a glance, a comment would be nice (i.e. It isn't immediately obvious why this fixes the bug)

Reply:
done